tools/subset.py

```diff
@@ -181,8 +181,6 @@ def subset_font(font_in, font_out, unicodes, opts):
 
 
 def getsubset(subset, font_in):
-    subsets = subset.split('+')
```

Comment:
Looking at the code below, my guess is that `if 'menu' in subset` should probably have been `if 'menu' in subsets` and so on.

Reply:
Yes, it probably was, but it works since subset is a string. 

Tried to make a minimal change (and tried to search for the original code, but to no avail).

Reply:
+1 for changing the code below (or leaving as is for now with a comment if you don't dare to change it).

This code has been unchanged since it's addition in 2015, so there is no history to dig up.

Anyway, it's pretty obvious that this was an oversight and only accidentally worked. Checking `subsets` below is more clear should be marginally faster, is more safe in theory (even though it shouldn't make any practical difference for real usecases.

Reply:
OK! I'll change it. Both ways should work and using `subsets` is indeed probably more clear.

Reply:
The `if 'lang' in subset` would be true if just `lang-ext` were in `subset`, but not now that we're checking `subsets` (which is correctly split). I don't know if this is a thing that might happen or if it's even going to be a problem, but this tool is internal anyway.

Reply:
Yes, you are correct. One can probably find such cases.